app/models/rent.rb

```diff
@@ -0,0 +1,5 @@
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+  validates :user, :book, presence: true
```

Comment:
Ok, and: `intial_date` and `final_date`

Reply:
I can't believe I keep missing these. I'll fix it now.